engine/sql/api/218_workflow_retention_policy.sql

```diff
@@ -0,0 +1,12 @@
+-- +migrate Up
+alter table "workflow" add column retention_policy TEXT;
+update workflow set retention_policy = 'return (git_branch_exist == "false" and run_days_before < 2) or run_days_before < 365';
```

Comment:
I wonder if we should keep NULL as default value and refers to the default rule stored in API configuration to be able to change the default rule for ALL the workflow easily